### PR TITLE
Update providers to version 5.0

### LINKED ISSUE #2919: Built for version updates
Since [EF Core 5.0](https://devblogs.microsoft.com/dotnet/announcing-the-release-of-ef-core-5-0/) is released, it would be good to update providers that support it (see below).  BTW links for Db2, Informix are dead.

Microsoft.EntityFrameworkCore.SqlServer
Microsoft.EntityFrameworkCore.Sqlite
Microsoft.EntityFrameworkCore.InMemory
Microsoft.EntityFrameworkCore.Cosmos
Npgsql.EntityFrameworkCore.PostgreSQL
Pomelo.EntityFrameworkCore.MySql (prerelease)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c9338b2a-93b1-595a-2e8e-86119e6eecbd
* Version Independent ID: 9890ab79-bc72-6075-4294-5450e6693668
* Content: [Database Providers - EF Core](https://docs.microsoft.com/en-us/ef/core/providers/?tabs=dotnet-core-cli)
* Content Source: [entity-framework/core/providers/index.md](https://github.com/dotnet/EntityFramework.Docs/blob/master/entity-framework/core/providers/index.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @ajcvickers
* Microsoft Alias: **avickers**

### PR DESCRIPTION
**Pull request by roji**
Closes #2919

### CODE REVIEW

**Comment by mguinness** [comment]
Pomelo.EntityFrameworkCore.MySql supports 5.0 in prerelease, can that be noted as such?

**Comment by roji** [comment]
@mguinness I'm assuming the 5.0 release is not far off, let's just update that at that point...

File: entity-framework/core/providers/index.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ Entity Framework Core can access many different databases through plug-in librar
...
-| [Npgsql.EntityFrameworkCore.PostgreSQL](https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL)                                                                         | PostgreSQL                      | [Npgsql Development Team](https://github.com/npgsql)                                            |                                            | 3.1               | [docs](https://www.npgsql.org/efcore/index.html)                                                                                               |
+| [Microsoft.EntityFrameworkCore.SqlServer](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.SqlServer)                                                                     | SQL Server 2012 onwards         | [EF Core Project](https://github.com/dotnet/efcore/) (Microsoft)                                |                                            | 5.0               | [docs](xref:core/providers/sql-server/index)                                                                                                   |
```
Comment (smitpatel) [inline entity-framework/core/providers/index.md:23]: Should we update everything to 5.0 removing 3.1 or should we leave 3.1 in? as 3.1 is LTS.

Comment (roji) [reply entity-framework/core/providers/index.md:23]: Hmm... Maybe we change the column title to "Latest supported version", and implicitly 5.0 means that 3.1 is also supported? Just to keep things simple and convey the important information.

**Review by bricelam — APPROVED** [review]

**Pull request merged**